Sprint-3/revise/implement/is-prime.js

```diff
@@ -0,0 +1,27 @@
+//Function should return true if a number is prime and false if it is not.
+
+function isPrime(num) {
+    const number = Number(num); //Ensures that input is a number
+
+    if (isNaN(number) || num === null || num === undefined) { //handles for non numbers, null and undefined inputs
+        throw new Error("Please enter a number");
+    }
+    
+    const integerNumber = Math.floor(number); //After checking for invalid inputs, it converts number inputs to an integer
+
+    if (integerNumber <= 1 ) {
+        return false; //1 is not a prime number
+    }
+
+    if (integerNumber <= 3) {
+        return true; //2 and 3 are prime numbers
+    }
+
+    if (integerNumber % 2 === 0 || integerNumber % 3 === 0) {
+        return false; //Numbers divisible by 2 or 3 with no remainder are prime numbers 
+    }
+
+    return true; //All other instances are prime numbers 
```

Comment:
#### Todo
5 is a prime numbers.
5 * 5 = 25 is not divisible by 2 or 3, but it is divisible by 5. So 25 is not a prime number.

You need additional code to make this function work for other integers.